ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/About.ey

```diff
@@ -0,0 +1,12 @@
+%e-yaml
+---
+
+Name: File Dropper
+Identifier: FileDroper
+Author: Samuel Venable
+Description: Allows the developer to drag and drop files and directories onto the game window and return their full paths.
+Default: false
+Icon: FileDropper.png
+
+Depends: None
```

Comment:
You forgot to put that the extension depends on the Win32 platform.

Reply:
I was going to do this initially but i didn't know whether to type Windows or Win32. I look at your direct show and several other windows-only extensions you and others wrote and looks like you guys forgot the same thing. I couldn't use those Windows only extensions as a reference for this so I just assumed if those got merged in their insufficient state that you wouldn't complain about this one either.

Reply:
Here: <https://github.com/enigma-dev/enigma-dev/blob/8940a00862b30c9388472c1b380edfc6bda56a64/ENIGMAsystem/SHELL/Audio_Systems/DirectSound/Info/About.ey#L13>

Use DirectSound as an example. Read the Wiki for clarification.
https://enigma-dev.org/docs/Wiki/About.ey

Reply:
Fixed it. Thank you!